src/screens/Box.js

```diff
@@ -561,6 +561,42 @@ const BoxPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="skeleton">
+          <Description>
+            Make the contents of the box 'skeletonized'. When this property is
+            supplied, any components within the Box that are skeleton-aware will
+            appear as gray boxes instead of their usual text or contents. This
+            is intended as a placeholder representation instead of the real
+            contents when it is still being loaded asynchronously. Some
+            skeleton-aware components are Box (when a border or background is
+            supplied), Text, Heading and Paragraph.
```

Comment:
Paragraph is mentioned here but I don't see skeleton prop Paragraph documentation in this PR.

Reply:
Paragraph doesn't have any skeleton-related props to document. It does react to being in a SkeletonContext however.